### DISCUSSION TITLE
`sls package` with layers without accesskeys

### DISCUSSION THREAD

**Discussion by fsojye** [serverless/serverless#9779] [Q&A] [answered]
How to package a serverless project without AWS access keys or permission?

Thanks!

**Comment by pgrzesik** [comment, marked as answer]
Hello @fsojeboroy - in general it should be possible to do with without access keys, unless your configuration depends on calling AWS during resolution e.g. because you use layers or `aws` variable source. Could you share the problematic configuration by any chance?

**Reply by fsojye** [reply]
you are right. I am using layers. I need to package the project including the layers but I dont have Cloudformation permission

**Reply by pgrzesik** [reply]
That's the behavior at the moment - here's the related issue: https://github.com/serverless/serverless/issues/8187

**Reply by fsojye** [reply]
Thanks @pgrzesik I may need to separate the layer packaging and deployment from the main proj
